templates/themes/app/content/views/payload/ng_contact_form.html.twig

```diff
@@ -15,14 +15,14 @@
         {{ form_errors(form) }}
 
         <fieldset>
-            {{ form_row(form.sender_first_name) }}
+            {{ form_row(form.sender_first_name, { ngparams: { autocomplete: 'ngsite.collected_info.given_name.attribute'|trans } })}}
```

Comment:
I believe you don't need to use `ngparams`. It looks like `ngparams` was used to override already existing `label` value with our own implementation (for which I'm not 100% clear why, btw). This should work by only transfering `autocomplete` variable.

Reply:
it doesn't work when I change it to` {{ form_row(form.sender_first_name, { autocomplete: 'ngsite.collected_info.given_name.attribute'|trans } })}`

Reply:
Alright then :+1: